irohad/main/impl/ordering_init.cpp

```diff
@@ -17,7 +17,7 @@
 
 #include "main/impl/ordering_init.hpp"
 #include "model/peer.hpp"
-#include "ordering/impl/ordering_service_transport_grpc.hpp"
+#include "ametsuchi/ordering_service_persistent_state.hpp"
```

Comment:
How it's work? Why you remove transport from here?

Reply:
duplicate in .hpp